### ISSUE TITLE
`docs=()` directive does not work

### ISSUE THREAD

**Issue by Nuc1eoN** [gobolinux/Compile#54]
The docs `docs=()` directive does not appear to do anything at all, unless I am mistaken.
It seems to be broken.

See package `GLM`.

**Comment by hishamhm** [comment]
> See package GLM.

I think it works, but it don't think it supports wildcards. 

Implementation in Functions/Compile looks like this:

``` bash
for i in COPYING LICENSE README* NEWS AUTHORS BUGS TODO COPYRIGHT ${docs:+"${docs[@]}"}
```

**Comment by Nuc1eoN** [comment]
Yes you are right, I corrected it in the GLM recipe https://github.com/gobolinux/Recipes/commit/3e937017a4cba69b018c54567d68f9714e54f0f2#diff-6f368c07c35092c010bf62ab020949084ec7604591d21cd9d4acb7dd44b4b360

I might open a feature request, since wildcards would be very useful. Actually I have seem quite some recipes that defined wildcards for docs=, so those are probably "broken".
